sdk/src/main/java/com/horizen/tools/utils/CommandProcessor.java

```diff
@@ -9,7 +9,7 @@ public CommandProcessor(MessagePrinter printer) {
         this.printer = printer;
     }
 
-    protected abstract void processCommand(String input) throws IOException;
+    protected abstract void processCommand(String input) throws Exception;
```

Comment:
Because it throws exception. I changed to IOException, but it expects Exception
public static FieldElement getInputKeysRootHash(SchnorrPublicKey[] signingKeys, SchnorrPublicKey[] masterKeys, long maxPks) throws Exception {
        return nativeKeysRootHash(signingKeys, masterKeys, maxPks);
    }